mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -1370,6 +1372,7 @@ impl TakerSwap {
         let send_abort_handle =
             broadcast_swap_message_every(self.ctx.clone(), swap_topic(&self.uuid), msg, 600., self.p2p_privkey);
 
+        Timer::sleep(20.).await;
```

Comment:
Is this required?

Reply:
IIRC, I added it as a workaround for some failing test. Will recheck and respond here.

Reply:
It's not required anymore. I think I did it when many of swap methods had dummy implementation and trade went too fast because of it causing failures on certain steps. Deleted.